cost/aws/old_snapshots/aws_delete_old_snapshots.pt

```diff
@@ -6,7 +6,7 @@ category "Cost"
 severity "low"
 default_frequency "daily"
 info(
-  version: "3.4",
+  version: "3.6",
```

Comment:
should this be v3.5? (based on my previous comment regarding the changelog file)